.jshintrc

```diff
@@ -17,7 +17,7 @@
      "undef": true, 
      "unused": true, 
      "trailing": true,
-     "maxparams": 3, 
+     "maxparams": 4,
```

Comment:
Travis CI was showing error for the function, composeSusiMessage, so changed the limit in jshintrc file.

Reply:
Do I need to change in `gruntfile` as well @gabru-md @zamhaq ?

Reply:
I think `travis CI` runs the `gruntfile` so it might have to be changed. Can we use any other approach instead of increasing the number of params? If not then let's increase the number of params. :smile:

Reply:
@zamhaq the problem is to track from which call the `speak` parameter has been changed. I think it can be solved using a global variable, I will try that approach ASAP.